Sources/ClientRuntime/Networking/Http/HttpContext.swift

```diff
@@ -181,11 +183,63 @@ public class HttpContextBuilder {
     /// - Returns: `self`, after the partition ID is set as specified.
     @discardableResult
     public func withPartitionID(value: String?) -> HttpContextBuilder {
-        self.attributes.set(key: partitionID, value: value)
+        self.attributes.set(key: AttributeKeys.partitionId, value: value)
+        return self
+    }
+
+    @discardableResult
+    public func withPath(value: String) -> HttpContextBuilder {
+        self.attributes.set(key: AttributeKeys.path, value: value)
+        return self
+    }
+
+    @discardableResult
+    public func withResponse(value: HttpResponse) -> HttpContextBuilder {
+        self.response = value
+        return self
+    }
```

Comment:
Its better if public methods have a comment explaining them like withPartitionId (line 183) -- apply across this PR

Reply:
Comments on `withPartitionId` as well as lack of comments on other methods were pre-existing. To write comment for each method would require investigation into what each field is really used for - I'll create a backlog ticket that includes both adding comments & possibly refactoring the getters to not use forced unwrapping (mentioned in other comment).

Reply:
Issue created: https://github.com/awslabs/aws-sdk-swift/issues/1168

Will pick up after finishing SRA Identity & Auth